public/components/datasources/components/manage/accelerations/__tests__/acceleration_table.test.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { AccelerationTable } from '../acceleration_table';
+
+import React from 'react';
+import { dummyAccelerations } from '../../../../../../../test/datasources';
+import * as Plugin from '../../../../../../plugin';
+
+describe('Acceleration Table Test', () => {
+  configure({ adapter: new Adapter() });
+
+  Plugin.getRenderAccelerationDetailsFlyout = jest.fn();
+
+  it('Render empty acceleration table', () => {
+    const wrapper = mount(<AccelerationTable accelerations={[]} />);
+
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Let's add more interaction tests for flyouts and table once we connect these with the backend

Reply:
+1, you can mock like this:

```
jest.mock('../../../../plugin', () => ({
  getRenderAccelerationDetailsFlyout: jest.fn(() =>
    jest.fn().mockImplementation(() => console.log('Acceleration Details Flyout Rendered'))
  ),
  getRenderAssociatedObjectsDetailsFlyout: jest.fn(() =>
    jest.fn().mockImplementation(() => console.log('Associated Objects Details Flyout Rendered'))
  ),
}));
```
 so that we can test the render:
 ```
  it('renders acceleration details correctly and triggers flyout on click', () => {
    const wrapper = mount(<AssociatedObjectsDetailsFlyout tableDetail={mockTableDetail} />);
    expect(wrapper.find('EuiInMemoryTable').at(0).find('EuiLink').length).toBeGreaterThan(0);

    wrapper.find('EuiInMemoryTable').at(0).find('EuiLink').first().simulate('click');
    expect(plugin.getRenderAccelerationDetailsFlyout).toHaveBeenCalled();
  });
 ```
Reference: https://github.com/opensearch-project/dashboards-observability/pull/1496/files#diff-9a67ab280ec2234577feebc0875415a357dcf4e0f6f4d264a96b18ea5264221eR32